src/server/c-interop-old.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: Native extensions for the standalone Dart VM
-description: Learn how to enable command-line Dart apps to call C/C++ functions.
-date: 2012-05-01
-category: dart-vm
+description: The original way for command-line Dart apps to call C/C++ functions.
```

Comment:
I think I would use something like 'Integrating Dart and native code using native extensions'

Also, I would prefer not to rename the file. Old carries connotations. Native-extensions.md is neutral.

Reply:
I'd like it to start with the same letters as the other page, so the other page will be selected in the nav when this one is current. Maybe... c-interop-extensions.

Reply:
`c-interop-native-extensions` would be ok